### ISSUE TITLE
Add example showing how it handles rounding big integers

### ISSUE THREAD

**Issue by nealmcb** [valdezt/sigfig#3]
E.g. is `round(123456, 4)` == `123500` or `123500.0` or something else?

I think the latter (a common result in various other approaches [as noted on stackoverflow](https://stackoverflow.com/a/48812729/507544)) is wrong since adding a decimal point is commonly interpreted as extending the precision.

(I know I could install it myself and find out, but others will want to know also, and I don't see it offhand in the docs.)
Thanks!

**Comment by MikeBusuttil** [comment]
agreed @nealmcb !  the result in this package the former, not the latter.  I'll review that stackoverflow post and consider adding it to the docs
